assets/js/main.js

```diff
@@ -52,3 +52,20 @@ $("#btn-modal-small").click(function() {
 $(".modal .close").click(function() {
   $(".modal").fadeOut();
 });
+
+
+// script to show and hide password
+$(".show-hide").click(function() {
+  $(this).toggleClass("ms-icon icon-hide");
```

Comment:
Some lines later it seems that you update the class again, something should be redundant, right?